### PR TITLE
Ensure cron jobs don't overlap in time

### PR DESCRIPTION
**Pull request by JoseKilo**
This change will lock the cron jobs on a file using `flock`.

### CODE REVIEW

File: tasks/setup_cron_tasks.yml [hunk 1]
```diff
@@ -10,10 +10,11 @@
     month: "{{ item.frequency.month | default('*') }}"
     weekday: "{{ item.frequency.weekday | default('*') }}"
     job:
```
Comment (farridav) [inline tasks/setup_cron_tasks.yml]: should this be optional?

File: tasks/setup_cron_tasks.yml [hunk 2]
```diff
@@ -10,10 +10,11 @@
...
+      flock -n {{ item.name|replace(' ', '_')|lower }}.lockfile /bin/bash -c '
```
Comment (farridav) [inline tasks/setup_cron_tasks.yml]: name could contain any number of bad chars.. e.g ampersand..

Comment (JoseKilo) [inline tasks/setup_cron_tasks.yml]: Relying on a `slug` field now. Added to the documentation

File: tasks/setup_cron_tasks.yml (hunk 1 above)
Comment (JoseKilo) [inline tasks/setup_cron_tasks.yml]: Done: optional and false by deafult

File: README.md [hunk 3]
```diff
@@ -128,6 +128,22 @@ the process running the cron, the command will look something like this:
 Note: you can set day, hour, minute or month in frequency, values not set
 default to '\*'
 
+You can also specify that your command shouldn't run twice at once, in that
```
Comment (robertweglarek) [inline README.md]: `run multiple times at once` ?

Comment (JoseKilo) [inline README.md]: Perfect wording ! fixed

**Pull request merged**